setup.sh

```diff
@@ -77,58 +101,40 @@ function _get_absolute_script_directory
     DIR="$(realpath -e -L "${0}")"
     DIR="${DIR%/setup.sh}"
   fi
-}
-
-DIR="$(pwd)"
-_get_absolute_script_directory
 
-CRI=
-CONFIG_PATH=
-CONTAINER_NAME=
-DEFAULT_CONFIG_PATH="${DIR}/config"
-IMAGE_NAME=
-INFO=
-USE_TTY=
-USE_SELINUX=
-VOLUME=
-WISHED_CONFIG_PATH=
-
-function _check_root
-{
-  if [[ ${EUID} -ne 0 ]]
-  then
-    echo "Curently, DMS doesn't support podman's rootless mode.
-Please run this script as root user."
-
-    exit 1
-  fi
+  DEFAULT_CONFIG_PATH="${DIR}/config"
 }
 
-function _update_config_path
+function _handle_config_path
 {
-  if [[ -n ${CONTAINER_NAME} ]]
+  if [[ -z ${WISHED_CONFIG_PATH} ]]
   then
-    VOLUME=$(${CRI} inspect "${CONTAINER_NAME}" \
-      --format="{{range .Mounts}}{{ println .Source .Destination}}{{end}}" | \
-      grep "/tmp/docker-mailserver$" 2>/dev/null || :)
-  fi
+    # no wished config path
+    if [[ -n ${CONTAINER_NAME} ]]
+    then
+      VOLUME=$(${CRI} inspect "${CONTAINER_NAME}" \
+        --format="{{range .Mounts}}{{ println .Source .Destination}}{{end}}" | \
+        grep "/tmp/docker-mailserver$" 2>/dev/null || :)
```

Comment:
Might be a bit OCD of me, but do you see any value in lifting `/tmp/docker-mailserver` up into a var at the top? (`DMS_CONFIG`?)

Reply:
I will have a look at all occurrences and decide :)

Reply:
> at all occurrences

Do not make a change to many files for that string. It'll cause big conflicts for a PR I need to push that modifies lines often with that value in it.

Reply:
I'll only do it for `setup.sh`.

Reply:
Done. [c006a977c55ff3e5fe90a540cd153372165e9363]